.travis.yml

```diff
@@ -17,14 +17,13 @@ matrix:
     fast_finish: true
     include:
           # Test with lowest dependencies
-        - php: 7.1
+        - php: 7.3
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
 
           # Test the latest stable release
-        - php: 7.1
-        - php: 7.2
         - php: 7.3
-          env: COVERAGE=true TEST_COMMAND="composer test-ci" DEPENDENCIES="henrikbjorn/phpspec-code-coverage:^1.0"
```

Comment:
This may need some discussion, but this dependency is [deprecated](https://github.com/henrikbjorn/PhpSpecCodeCoverageExtension) so I decided to remove code coverage.

Reply:
fine for me